apps/web/src/content/blog/2026-03-01.md

```
@@ -0,0 +1,82 @@
+
+- **ホーム**: ヒーロー・スキル概要・今やっていること
+- **プロフィール**: タイムライン・スキル分類（得意 / 実務経験あり / 触ったことがある）・趣味・PC スペック
+- **Works**: 制作物ポートフォリオ
+- **Blog**: Markdown 記事 + タグによるクライアントサイドフィルタリング
+- **訪問者カウンター**: サーバーレス API で集計、フッターに表示
+- **ダーク / ライトモード**: `prefers-color-scheme` + CSS カスタムプロパティで自動切替
+- **レスポンシブデザイン**: モバイル〜 PC 対応
+- **SEO**: OG タグ・canonical URL
+- **アクセシビリティ**: スキップリンク・ARIA ラベル
+
+## これから
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)